frameworks/cassandra/universe/config.json

```diff
@@ -55,6 +55,11 @@
           "description": "Enable DC/OS Virtual Networking",
           "type": "boolean",
           "default": false
+        },
+        "log_level": {
```

Comment:
Yeah, let's do an enum with all options. I'd rather not suffer no logs due to fat fingering.


OFF
FATAL
ERROR
WARN
INFO
DEBUG
TRACE
ALL

from: https://logging.apache.org/log4j/2.x/manual/customloglevels.html